android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -257,10 +261,56 @@ private void loadWebView() {
                 setServerBasePath(path);
             }
         }
+
+        if (!this.isMinimumWebViewInstalled()) {
+            String errorUrl = this.getErrorUrl();
+            if (errorUrl != null) {
+                webView.loadUrl(errorUrl);
+                return;
+            } else {
+                Logger.error(MINIMUM_ANDROID_WEBVIEW_ERROR);
+            }
+        }
+
         // Get to work
         webView.loadUrl(appUrl);
     }
 
+    @SuppressLint("WebViewApiAvailability")
+    private boolean isMinimumWebViewInstalled() {
+        PackageManager pm = getContext().getPackageManager();
+
+        // Check getCurrentWebViewPackage() directly if above Android 8
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            PackageInfo info = WebView.getCurrentWebViewPackage();
+            String majorVersionStr = info.versionName.split("\\.")[0];
+            int majorVersion = Integer.parseInt(majorVersionStr);
+            return majorVersion >= MINIMUM_ANDROID_WEBVIEW_VERSION;
+        }
+
+        // Otherwise manually check WebView versions
+        try {
+            PackageInfo info = pm.getPackageInfo("com.google.android.webview", 0);
+            String majorVersionStr = info.versionName.split("\\.")[0];
+            int majorVersion = Integer.parseInt(majorVersionStr);
+            return majorVersion >= MINIMUM_ANDROID_WEBVIEW_VERSION;
+        } catch (Exception ex) {
+            Logger.warn("Unable to get package info for 'com.google.android.webview'", ex.toString());
+        }
+
+        try {
+            PackageInfo info = pm.getPackageInfo("com.android.webview", 0);
+            String majorVersionStr = info.versionName.split("\\.")[0];
+            int majorVersion = Integer.parseInt(majorVersionStr);
+            return majorVersion >= MINIMUM_ANDROID_WEBVIEW_VERSION;
+        } catch (Exception ex) {
+            Logger.warn("Unable to get package info for 'com.android.webview'", ex.toString());
```

Comment:
```suggestion
            Logger.warn("Unable to get package info for 'com.android.webview'");
```
Logger.warn accepts tag + message or message, so you are passing the message as tag
Either remove the ex.toString() or concat it to the message instead of being an additional param